test/app/components/SaveControl.spec.js

```diff
@@ -31,27 +31,32 @@ describe('Export page tests', () => {
     sandbox.restore();
   });
 
-  it('Call remember article on toggle', () => {
+  it('Call “remember article” on toggle', () => {
```

Comment:
Please, don't use typographic quotes in the source code. I know it looks fancy but there are some arguments against:
1. Most people won't type them, so later the code will have different styles.
2. As most people don't have such quotes, they can't search the code effectively.
3. They are not ASCII, which might cause nasty problems. (though, unlikely.)